libmscore/scorefile.cpp

```diff
@@ -405,6 +405,7 @@ bool MasterScore::saveFile()
 
       QString name(info.filePath());
       QString basename(info.fileName());
+      // QString dirString = info.path();
```

Comment:
Why?

Reply:
dirString is used for tracing the process of turning it into backupDirString, it's just for testing. Must this note be taken down? Because maybe someone can later use it (when de-noted)...

Reply:
It isn't used anywhere else in the code, so just remove that line